cmd/ctlog/createctconfig/main.go

```diff
@@ -230,3 +206,33 @@ func mustMarshalAny(pb proto.Message) *anypb.Any {
 	}
 	return ret
 }
+
+// createConfigWithKeys creates otherwise empty CTLogCOnfig but fills
+// in PrivKey, and PubKey. Can not be used as is, but use it to construct
+// the base to build upon
+func createConfigWithKeys(ctx context.Context, keytype string) (*ctlog.CTLogConfig, error) {
+	var privKey crypto.PrivateKey
+	var err error
+	if *keyType == "rsa" {
+		privKey, err = rsa.GenerateKey(rand.Reader, bitSize)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to generate Private RSA Key: %w", err)
+		}
+	} else {
+		privKey, err = ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Like RSA's bit size, we should make the elliptic curve configurable. this will save us time in the future if we need stronger keys. we can just support p256, p384, and p521.